scss/_variables.scss

```diff
@@ -1710,25 +1712,28 @@ $tooltip-arrow-color:               null !default; // Deprecated in Boosted 5.2.
 // scss-docs-start popover-variables
 $popover-font-size:                 $font-size-sm !default;
 $popover-font-weight:               $font-weight-bold !default; // Boosted mod
-$popover-bg:                        $gray-400 !default; // Boosted mod: instead of `var(--#{$prefix}body-bg)`
+$popover-bg:                        $white !default; // Boosted mod: instead of `var(--#{$prefix}body-bg)`
 $popover-max-width:                 17.25rem !default;
 $popover-border-width:              var(--#{$prefix}border-width) !default;
-$popover-border-color:              $popover-bg !default; // Boosted mod
+$popover-border-color:              $gray-500 !default; // Boosted mod
 $popover-border-radius:             var(--#{$prefix}border-radius-lg) !default;
 $popover-inner-border-radius:       calc($popover-border-radius - $popover-border-width) !default; // stylelint-disable-line function-disallowed-list
 $popover-box-shadow:                null !default;
 
 $popover-header-font-size:          $font-size-base !default;
 $popover-header-bg:                 $popover-bg !default; // Boosted mod: instead of `var(--#{$prefix}secondary-bg)`
 $popover-header-color:              $headings-color !default;
-$popover-header-padding-y:          $spacer * .5 !default;
```

Comment:
After your [other comment](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1803#discussion_r1092925046), I I reintroduce `$popover-header-padding-y` in our code.

Since `$popover-body-padding-y: $popover-header-padding-y !default;` we have to keep it too. Removing or deprecating not needed anymore.